Tests/FoundationTests/CalendarExtensionTest.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CalendarExtension.swift
+//  SwifterSwift
+//
+//  Created by Chaithanya Prathyush on 09/11/17.
+//  Copyright © 2017 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+class CalendarExtensionTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    func testNumberOfDaysInAMonth() {
+        let calendar = Calendar.current
```

Comment:
Please update this to make sure tests always pass for people having other calendars :)
```swift
let calendar = Calendar(identifier: .gregorian)
```